cli/src/main/java/com/devonfw/tools/ide/io/PathCopyListener.java

```diff
@@ -0,0 +1,27 @@
+package com.devonfw.tools.ide.io;
+
+import java.nio.file.Path;
+
+/**
+ * Interface for a listener of {@link Path} while {@link FileAccess#copy(Path, Path, FileCopyMode, PathCopyListener) copying}.
+ *
+ * @see FileAccess#copy(Path, Path, FileCopyMode, PathCopyListener)
+ */
+@FunctionalInterface
+public interface PathCopyListener {
+
+  /**
+   * An empty {@link PathCopyListener} instance doing nothing.
+   */
+  static PathCopyListener NONE = (s, t, d) -> {
```

Comment:
??? Please elaborate.

Reply:
I do not get the point here? What are you suggesting?
This is an empty implementation that does nothing and it is used for the existing usages that did not have the ability for such a listener before. The JavaDoc is explaining that this is empty and the implementation is obviously empty. So what is the problem?

Reply:
Will also be discussed tomorrow.